### ISSUE TITLE
what does gamma: 0.5 in config/training/cpc.ymal mean?

### ISSUE THREAD

**Issue by Hu-chengyang** [Wendison/VQMIVC#30]
Dear PHD Wang:
I wonder what does gamma: 0.5 mean? Loss weight?

**Comment by Hu-chengyang** [comment]
My recon loss is 0.892, I wonder if there are some approaches to reduce it?

**Comment by Wendison** [comment]
> Dear PHD Wang: I wonder what does gamma: 0.5 mean? Loss weight?

No, it is the learning rate decaying weight as shown here: https://github.com/Wendison/VQMIVC/blob/f470ca8edfc4d2b7faa8c87fd838b0d7203ccf28/scheduler.py#L34



> My recon loss is 0.892, I wonder if there are some approaches to reduce it?

According to my experiments on VCTK, reconstruction loss should be below 0.5. I'm not sure about what dataset you used, if you want to reduce the loss, you may need to tune the parameters of networks, e.g., layer number/dimension. One straightway to verify whether your training tends to be successful is to listen to the intermediate converted results, i.e., generate the converted wavs in the middle of training process.

**Comment by Hu-chengyang** [comment]
thank you very much
